main_logic/tts_client.py

```
@@ -1959,452 +2139,159 @@ def minimax_tts_worker(request_queue, response_queue, audio_api_key, voice_id, b
-                audio_chunk_buffer.clear()
-
-        async def _wait_for_event(target_event: asyncio.Event, timeout_s: float) -> bool:
-            target_task = asyncio.create_task(target_event.wait())
-            disconnect_task = asyncio.create_task(disconnect_event.wait())
-            try:
-                done, pending = await asyncio.wait(
-                    {target_task, disconnect_task},
-                    timeout=timeout_s,
-                    return_when=asyncio.FIRST_COMPLETED,
+        text_buffer = ""
+        # 有序合成任务队列：保证音频按文本顺序输出
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)